test/ftest/src/search.erl

```diff
@@ -0,0 +1,19 @@
+-module(search).
```

Comment:
`search` is a very general name for this test module; I suggest you change it to a more specific one.

More importantly, I suggest you add some comment describing what this test module is supposed to be testing.

Reply:
Done.